routers/web/org/setting.go

```diff
@@ -251,3 +230,28 @@ func SettingsRenamePost(ctx *context.Context) {
 	ctx.Flash.Success(ctx.Tr("org.settings.rename_success", oldOrgName, newOrgName))
 	ctx.JSONRedirect(setting.AppSubURL + "/org/" + url.PathEscape(newOrgName) + "/settings")
 }
+
+// SettingsChangeVisibilityPost response for change organization visibility
+func SettingsChangeVisibilityPost(ctx *context.Context) {
+	visibility := structs.VisibilityModes[ctx.FormString("visibility")]
+	if !visibility.IsValid() {
```

Comment:
`IsValid` is not needed.

You can just use `visibility, ok := structs.VisibilityModes[ctx.FormString("visibility")]`

Reply:
d1f1cf8a29